ivy/functional/frontends/jax/numpy/manipulations.py

```diff
@@ -151,3 +151,49 @@ def vsplit(ary, indices_or_sections):
 @to_ivy_arrays_and_back
 def hsplit(ary, indices_or_sections):
     return ivy.hsplit(ary, indices_or_sections)
+
+
+@to_ivy_arrays_and_back
+def bartlett(M):
+    if M == 1:
+        return ivy.ones(1)
+    elif M % 2 == 0:
+        return ivy.concatenate(
```

Comment:
`ivy.concatenate` is not present in ivy API but, it is available as `ivy.concat`